tests/feat-229-override-annotationbased-config/src/test/java/io/dekorate/examples/ComponentSpringBootExampleTest.java

```diff
@@ -47,14 +47,10 @@ public void shouldContainComponentAndLink() {
     // This may be null during the release process where HEAD point to a commit instead of a branch.
     //assertNotNull("", component.getSpec().getBuildConfig().getRef());
     assertEquals(DeploymentMode.build, component.getSpec().getDeploymentMode());
-    Link link = (Link) items.get(1);
-    Assertions.assertEquals("Link", link.getKind());
-    Env[] envs = link.getSpec().getEnvs();
-    Assertions.assertEquals(1, envs.length);
-    Assertions.assertEquals("key1-from-properties", envs[0].getName());
-    Assertions.assertEquals("val1-from-properties", envs[0].getValue());
-    Assertions.assertEquals("hello-world", link.getMetadata().getName());
-    Assertions.assertEquals("target", link.getSpec().getComponentName());
+    Map<String, String> labels = component.getMetadata().getLabels();
```

Comment:
👍
I had added a comment on one previous change that removed a test of property overriding, so good job re-adding it elsewhere!